fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -27,6 +27,10 @@ int32_t ineg(int32_t l) {
     return -l;
 }
 
+int64_t lneg(int64_t l){
```

Comment:
Space before brace missing here.

Reply:
@0xdaryl Do I need to rename my PR name by omitting "Aarch64" as I have implemented the triltest for lneg?

Reply:
Yes, PR and commit titles are separate. I've amended your PR comment however to include the body of your commit.